examples/matlab/parabolic1D.m

```diff
@@ -50,7 +50,8 @@
     tic
     % Implicit
     L = -alpha*dt*L + speye(size(L));
-    
+    dL=decomposition(L); 
+    %doing decomposition to speed up the solving the systems
```

Comment:
This needs to be better. Why/how does it speed up the solution? Almost no one does this, and it would be nice to explain why this works.

Reply:
The comment needs to explain better what "decomposition" does.

Reply:
Hello Jared,
The above commit fixes all of the above issues you had suggested with the style of my code.